src/gui/addnewtorrentdialog.h

```diff
@@ -38,6 +38,7 @@
 
 #include "base/bittorrent/infohash.h"
 #include "base/bittorrent/torrentinfo.h"
+#include "base/bittorrent/session.h"
```

Comment:
Or we can just get rid of this member. In fact we don't need it at all. We just use a parameter of this type to initialize the dialog, and then just create the object of this type for passing to the Session. Why we need to store it?